src/frontend/screens/Settings/AboutMapeo.js

```diff
@@ -87,6 +95,13 @@ const AboutMapeo = () => {
         />
         <DeviceInfoListItem label={t(m.androidBuild)} deviceProp="buildId" />
         <DeviceInfoListItem label={t(m.phoneModel)} deviceProp="model" />
+        <DeviceInfoListItem label={t(m.deviceId)} deviceProp="deviceId" />
+        <ListItem>
+          <ListItemText
+            primary={t(m.deviceId)}
+            secondary={displayDeviceId}
+          ></ListItemText>
+        </ListItem>
```

Comment:
list item here doesn't seem necessary if you have the DeviceInfoListItem already

Reply:
ah got it 👍

Reply:
Looks like we need a better name. 

This is the public key of the device's writeable hypercore. It is useful for debugging when trying to establish which observations were created on which device. Useful when eg you are analysing a Mapeo project and find missing sparse data in particular hypercore and figuring out which device that is from.

Reply:
got it! makes sense after reading the original PR description. few suggestions that come off the top of my head in case any may come relatively close:

- Device Core ID
- Device Data ID
- Device Hypercore ID (because why not lol)
- Core ID
- Hypercore ID
- Core Data ID